doc/dftb+/manual/dftbp.tex

```diff
@@ -676,7 +677,7 @@ \subsection{SecondDerivatives\cb}
 \label{sec:dftbp.SecondDerivatives}
 
 Calculates the second derivatives\index{Hessian} of the energy and the
-Born charges (currently both only using a numerical first
+dipole and polarizability (currently using a numerical first
```

Comment:
Could you add a reference to the modes-section, so that people know which tool can process them within the DFTB+ universe?

Reply:
done